### PR TITLE
Pass ban-list to block malicious nodes

### PR DESCRIPTION
**Pull request by sethsimmons**
This change requires https://github.com/btcpayserver/dockerfile-deps/pull/18 to be merged first, or else the Docker container will fail to start.

### CODE REVIEW

**Comment by sethsimmons** [comment]
Please ensure https://github.com/btcpayserver/dockerfile-deps/pull/18 is merged before merging this, and would love some more testing as I do not personally use BTCPayServer normally.

**Comment by astupidmoose** [comment]
FYI: I have https://github.com/btcpayserver/btcpayserver-docker/pull/412 ready to be merged for .7 but this file assumes .6

Docker-deps was already updated for .7, so your reference might be outdated. If you make the changes on this file to adjust to .7, I can likely close my PR.

**Comment by sethsimmons** [comment]
Bumped it to v0.17.1.7, nice catch.

File: docker-compose-generator/docker-fragments/monero.yml [hunk 1]
```diff
@@ -4,16 +4,16 @@ services:
   monerod:
     restart: unless-stopped
     container_name: btcpayserver_monerod
-    image: btcpayserver/monero:0.17.1.6
-    entrypoint: monerod --rpc-bind-ip=0.0.0.0 --confirm-external-bind --rpc-bind-port=18081 --non-interactive --block-notify="/bin/sh ./scripts/notifier.sh -X GET http://btcpayserver:49392/monerolikedaemoncallback/block?cryptoCode=xmr&hash=%s" --hide-my-port --prune-blockchain
+    image: btcpayserver/monero:0.17.1.7
+    entrypoint: monerod --rpc-bind-ip=0.0.0.0 --confirm-external-bind --rpc-bind-port=18081 --non-interactive --block-notify="/bin/sh ./scripts/notifier.sh -X GET http://btcpayserver:49392/monerolikedaemoncallback/block?cryptoCode=xmr&hash=%s" --hide-my-port --prune-blockchain --ban-list=/home/monero/.bitmonero/block.txt
```
Comment (astupidmoose) [inline docker-compose-generator/docker-fragments/monero.yml:8]: the ban list should reference:

`--ban-list=/home/monero/block.txt`

as per my comment in the deps commit.

**Pull request merged**